pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -86,6 +86,13 @@ const (
                            ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
 )
 
+// PrepareClickHouseConnection is used for unit testing
+var (
+	PrepareClickHouseConnection = func(input ClickHouseInput) (string, *sql.DB, error) {
+		return PrepareConnection(input)
+	}
+)
```

Comment:
it's a bit clunky that this needs to be an exported variable, but I understand that there isn't any obvious alternative here...